src/bin/dolos/common.rs

```diff
@@ -62,6 +69,19 @@ pub fn open_data_stores(config: &crate::Config) -> Result<Stores, Error> {
 pub fn setup_tracing(config: &LoggingConfig) -> miette::Result<()> {
     let level = config.max_level;
 
+    let exporter = LogExporter::builder()
+        .with_http()
+        .with_endpoint("http://localhost:4318/v1/logs")
+        .with_protocol(Protocol::HttpJson)
+        .build().unwrap();
+
+    // Create a new OpenTelemetry trace pipeline that prints to stdout
+    let provider = LoggerProvider::builder()
+        // .with_simple_exporter(exporter)
```

Comment:
using `with_simple_exporter` is usually better for dev environments (it's significantly more responsive)